### ISSUE TITLE
setup.py --record generates wrong paths when .separate_namespace used

### ISSUE THREAD

**Issue by neirbowj** [Legrandin/pycryptodome#119]
# Test case

```sh
stagedir=`mktemp -d`
recordfile=`mktemp`
touch .separate_namespace
python3.6 setup.py install \
    --single-version-externally-managed \
    --root=$stagedir \
    --prefix=/usr/local \
    --record=$recordfile
head $recordfile                     # <--- Look at output from this command
rm $recordfile && unset recordfile
rm -fr $stagedir && unset stagedir
```

Note that this is reproducible with versions of Python other than 3.6. I ran the test with Python 3.6.4 and setuptools 38.2.5. Other details of the environment available upon request.

# Expected output

```
/usr/local/lib/python3.6/site-packages/Cryptodome/__init__.py
/usr/local/lib/python3.6/site-packages/Cryptodome/Cipher/_mode_ocb.py
/usr/local/lib/python3.6/site-packages/Cryptodome/Cipher/Blowfish.py
/usr/local/lib/python3.6/site-packages/Cryptodome/Cipher/ChaCha20.py
/usr/local/lib/python3.6/site-packages/Cryptodome/Cipher/PKCS1_OAEP.py
/usr/local/lib/python3.6/site-packages/Cryptodome/Cipher/ARC2.py
/usr/local/lib/python3.6/site-packages/Cryptodome/Cipher/CAST.py
/usr/local/lib/python3.6/site-packages/Cryptodome/Cipher/__init__.py
/usr/local/lib/python3.6/site-packages/Cryptodome/Cipher/Salsa20.py
/usr/local/lib/python3.6/site-packages/Cryptodome/Cipher/ARC4.py
```

# Observed output

```
/usr/local/lib/python3.6/site-packages/Crypto/__init__.py
/usr/local/lib/python3.6/site-packages/Crypto/Cipher/_mode_ocb.py
/usr/local/lib/python3.6/site-packages/Crypto/Cipher/Blowfish.py
/usr/local/lib/python3.6/site-packages/Crypto/Cipher/ChaCha20.py
/usr/local/lib/python3.6/site-packages/Crypto/Cipher/PKCS1_OAEP.py
/usr/local/lib/python3.6/site-packages/Crypto/Cipher/ARC2.py
/usr/local/lib/python3.6/site-packages/Crypto/Cipher/CAST.py
/usr/local/lib/python3.6/site-packages/Crypto/Cipher/__init__.py
/usr/local/lib/python3.6/site-packages/Crypto/Cipher/Salsa20.py
/usr/local/lib/python3.6/site-packages/Crypto/Cipher/ARC4.py
```

**Comment by Legrandin** [comment]
Thanks. I can reproduce it. My guess is that the ``--root`` parameter is not handled correctly.

**Comment by Legrandin** [comment]
Fixed in 3.4.8
